.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on: [push, pull_request]
```

Comment:
Do we need this on both push and pull_request?

Reply:
Ah, no, good call. I think I should copy the `on:` field from #1733